src/main/java/edu/kis/powp/jobs2d/command/letters/LetterFactory.java

```diff
@@ -0,0 +1,33 @@
+package edu.kis.powp.jobs2d.command.letters;
+
+import edu.kis.powp.jobs2d.drivers.DriverManager;
+
+public class LetterFactory implements AbstractFactory<ComplexCommand> {
+
+    private static final String lLetter = "l";
+
+    private static final String uLetter = "u";
+
+    private static final String fLetter = "f";
+
+    private DriverManager driverManager;
+
+    public LetterFactory(DriverManager driverManager) {
+        this.driverManager = driverManager;
+    }
+
+    @Override
+    public ComplexCommand create(String letterCode) {
+        switch (letterCode) {
```

Comment:
This class should be closed for modification. 
You can use Map to keep factories, but new should not be called directly (e.g. factory method).
LetterFactory should be configured externally (maybe add registerLetterFactory method?)